deploy/rbac.yaml

```diff
@@ -35,7 +35,11 @@ rules:
   - replicasets
   verbs:
   - "*"
-
+- apiGroups: ["apps"]
+  resources:
+  - deployments/finalizers
```

Comment:
Based on the logs message and [this](https://github.com/operator-framework/operator-sdk/issues/1736#issuecomment-517806900) maybe you'd want to add services/finalizers RBAC as well?